nvchecker_source/maven.py

```diff
@@ -0,0 +1,56 @@
+# MIT licensed
+# Copyright (c) 2025 Dani Rodríguez <dani@danirod.es>, et al.
+
+from nvchecker.api import (
+    Entry,
+    KeyManager,
+    AsyncCache,
+    VersionResult,
+    GetVersionError,
+    HTTPError,
+    session,
+)
+import pathlib
+from xml.etree import ElementTree
+
+
+MAVEN_CENTRAL = "https://repo1.maven.org/maven2"
+
+
+async def get_version(
+    name: str,
+    conf: Entry,
+    *,
+    cache: AsyncCache,
+    keymanager: KeyManager,
+    **kwargs,
+) -> VersionResult:
+    # get project coordinates
+    group_id, artifact_id = conf.get("group"), conf.get("artifact")
+    if not group_id:
+        raise GetVersionError("Group identifier is not given")
+    if not artifact_id:
+        raise GetVersionError("Artifact identifier is not given")
+    group_id = group_id.replace(".", "/")
+
+    # build the URL according to the transformation rules
+    # -> https://maven.apache.org/repositories/layout.html
+    repo = conf.get("repo", MAVEN_CENTRAL)
+    repo_root = pathlib.PurePosixPath(repo)
```

Comment:
This turns `https:/repo1.maven.org/maven2` to `https:/repo1.maven.org/maven2`, which isn't accepted by those http clients.

Reply:
Yeah, it seems that was the reason. I tried to play it safe to concat the URL based on what I've seen on other sources, but I did not notice that the protocol was wrong.